packages/cli/src/cmds/beacon/options.ts

```diff
@@ -127,6 +127,7 @@ export const beaconExtraOptions: CliCommandOptions<BeaconExtraArgs> = {
 
   persistNetworkIdentity: {
     description: "Whether to reuse the same peer-id across restarts",
+    default: true,
```

Comment:
why?

Reply:
For a given ENR the cgc should persist across restarts.  But there is no spec saying a vm should restart as the same ENR to my knowledge (could be wrong about that though)....

I tend to think this should be true but i guess it doesn't have to be.  

Like we are pretty good peers now and that flag was added when that was not true so we could restart as a "new" node and not suffer the penalties from a previous run.  

I'm open to either option though.  curious what @philknows @twoeths @nazarhussain and @ensi321 think though

Reply:
I'm not seeing anything [in the specs](https://github.com/ethereum/consensus-specs/blob/master/specs/fulu/p2p-interface.md#enr-structure) that says you _need_ to retain the previous ENR on restart. But as long as you advertise the same `cgc` in the new ENR you should be fine though? 

I guess, what are the side-effects of this to discoverability or retention of good peers? If it disrupts sync/peering, maybe I'd lean towards retaining the same ENR per restart... but otherwise, I do appreciate the default privacy of switching ENRs often.

Reply:
yeah we should definitely highlight that in the release notes,  I guess having the option to set `--persistNetworkIdentity false` is a good way to reset the cgc without deleting the database

maybe we need to document that behavior somewhere, possible the CLI flag description

Reply:
updated in #8159